src/scriptdev2/scripts/kalimdor/durotar.cpp

```diff
@@ -33,53 +33,208 @@ EndContentData */
 
 enum
 {
-    SAY_PEON_AWAKE_1        = -1000795,
-    SAY_PEON_AWAKE_2        = -1000796,
+    SAY_PEON_AWOKEN = -1000795,
 
-    SPELL_PEON_SLEEP        = 17743,
-    SPELL_AWAKEN_PEON       = 19938,
+    SOUND_PEON_WORK = 6197,
+    SOUND_PEON_AWAKE_1 = 6292,
+    SOUND_PEON_AWAKE_2 = 6294,
 
-    NPC_SLEEPING_PEON       = 10556,
-    GO_LUMBERPILE           = 175784,
+    SPELL_PEON_SLEEP = 17743,
+    SPELL_AWAKEN_PEON = 19938,
+
+    NPC_SLEEPING_PEON = 10556,
+    GO_LUMBERPILE = 175784,
 };
 
 struct npc_lazy_peonAI : public ScriptedAI
 {
     npc_lazy_peonAI(Creature* pCreature) : ScriptedAI(pCreature)
     {
         Reset();
-        m_uiStopSleepingTimer = urand(30000, 120000);       // Set on spawn to a potential small timer, to get nice results for initial case
+        //tree waypoint
+        //TODO do this via invisible creatures once they're added in the db (or actual waypoints)
+        switch (m_creature->GetGUIDLow())
+        {
```

Comment:
Yea I think I'm just going to start ignoring you entirely from now on. If you had read the TODO that's literally *in* the excerpt you posted you would know that this is a temporary solution that I used because I couldn't get creature_movement to work.

If you know how to get creature_movement to work for the tree positions you could use that knowledge to make an actual contribution instead of uselessly yelling.

Also, IIRC the GUID's are sniffed so they shouldn't change. And I honestly don't see how the DB provider would suddenly change these. It's not like it's a third party, it's part of the same larger project.